mesheryctl/pkg/utils/error.go

```diff
@@ -128,7 +128,7 @@ func formatError(msg string, cmd cmdType) string {
 
 func ErrAttachAuthToken(err error) error {
 	return errors.New(ErrAttachAuthTokenCode, errors.Alert, []string{err.Error()},
-		[]string{"Authentication token not found. please supply a valid user token with the --token (or -t) flag. or login with `mesheryctl system login`"}, []string{}, []string{})
+		[]string{"Authentication token not found. please supply a valid user token using `mesheryctl system config [aks|eks|gke|minikube] -t <path>`. or login with `mesheryctl system login`"}, []string{}, []string{})
```

Comment:
I think we have the token flag for quite a few methods. Have we changed that? This seems a generic error type, not specific to `mesheryctl system config`

Also not sure if we should delete the `--token (or -t)` suggestion

Reply:
the idea here was not to delete the --token flag but to provide the full command i.e. `mesheryctl system config  [aks|eks|gke|minikube] -t <path>` which inturn will help the user in providing the token via this command. 

so the --token suggestion is not deleted but is intact to the helping command.

Reply:
So your command is an example right? Like this auth error comes on other commands as well right. Is it right to just specify one example?

Reply:
An example or a case, the case here considered is 
user is trying to list the pattern using `mesheryctl pattern list` but he is not authenticated. 
So a token is being asked from him to supply via the `mesheryctl system config [aks|eks|gke|minikube] -t <path>` cmd  or login with `mesheryctl system login` .

Reply:
I don't think a specific case should be mentioned for a generic authentication error type. 

@revolyssup

Reply:
Ok @alphaX86 @hershd23 thank you for the feedback, I will change this accordingly 🙂 no sweat.